src/components/SongRecommendation/SongRecommendation.css

```diff
@@ -0,0 +1,4 @@
+.songImage{
+    height: 100px;
+    width: 100px;
+}
```

Comment:
See how the images are a little squished? 

![image](https://user-images.githubusercontent.com/10343470/143284401-4e5376a0-5bb3-4154-b7fe-6967670d43c0.png)

Try this:

```suggestion
.songImage{
    height: 100px;
    width: 100px;
    padding-left: 0;
    padding-right: 0;
}
```
To remove the inherited padding from the `.row>*` rule:

![image](https://user-images.githubusercontent.com/10343470/143284368-d000fd4a-da08-4375-abc1-8283b2548dd0.png)